README.md

```diff
@@ -0,0 +1,235 @@
+# BookShop
+
+BookShop is an online store that allows users to browse and purchase books. It supports book categories, user roles, shopping carts, and efficient order management.
+
+## Video Demo
+
+[![Watch the demo video](images/video-thumbnail.png)](https://www.loom.com/share/a914d2d093be4c5f8fbda0196265d8ed?sid=53254b16-4466-425e-ad05-5c2ba0367705)
+
+Click the image above to watch the demo video.
+
+## Project Description
+
+The BookShop app includes the following domain models:
+- **User**: Contains information about registered users, including authentication and personal details.
+- **Role**: Represents user roles such as admin or user.
+- **Book**: Represents books available in the store.
+- **Category**: Represents categories for books.
+- **ShoppingCart**: Represents a user's shopping cart.
+- **CartItem**: Represents an item in a user's shopping cart.
+- **Order**: Represents orders placed by users.
+- **OrderItem**: Represents an item in a user's order.
+
+## Technologies Used
+
+### Spring Boot
+Spring Boot was used to create a stand-alone, production-grade Spring-based application with minimal configuration. It facilitated rapid development and deployment.
+
+### Spring Security
+Spring Security was implemented to provide robust authentication and authorization mechanisms. It ensured that only authorized users could access certain endpoints, and it handled login and registration processes securely.
```

Comment:
can we make them as a bulleted list instead of topics?
```suggestion
* **Spring Boot**: Spring Boot was used to create a stand-alone, production-grade Spring-based application with minimal configuration. It facilitated rapid development and deployment.

* **Spring Security**: Spring Security was implemented to provide robust authentication and authorization mechanisms. It ensured that only authorized users could access certain endpoints, and it handled login and registration processes securely.
```